pkg/controller/tls/certs.go

```diff
@@ -129,20 +129,44 @@ func ProvisionCerts(req router.Request, resp router.Response) error {
 		return err
 	}
 
-	// FIXME: use error list instead of exiting on first error
-	for _, pb := range appInstance.Spec.Ports {
-		// Skip: name empty, not FQDN, already covered by on-acorn.io
-		if pb.ServiceName == "" || len(validation.IsFullyQualifiedDomainName(&field.Path{}, pb.ServiceName)) > 0 || strings.HasSuffix(pb.ServiceName, "on-acorn.io") {
+	provisionedCerts := map[string]interface{}{}
+	var errs []error
+
+	for i, ep := range appInstance.Status.Endpoints {
```

Comment:
Is programming the certs based off of the status field ok/acceptable @ibuildthecloud ?

Reply:
Yep